### PR TITLE
Features/14 contact page

### LINKED ISSUE #14: Front - Page de contact
## User stories
Un utilisateur doit pouvoir accéder a la page de contact dans le footer, il retrouvera sur cette page des informations essentiels (adresse, email, numéro de téléphone etc...) et un formulaire de contact avec comme champ : 
- sujet de l'email
- objet de l'email
- corps de texte/description 
- bouton d'envoi 

Une fois complété, il a une notification en cas de succès ou non et recevra un email de confirmation de réception.

### Contrainte 
Afficher les erreurs de formulaires sur chaque input

### Idées
- Mettre une carte

## Tasks
* [x] Création du formulaire
* [x] Création de la page de contact avec mise en forme (voir maquette)
* [x] Gérer l'affichage des erreurs et des redirections 
* [x] Affichage des notifications une fois le formulaire soumis

### PR DESCRIPTION
**Pull request by Dercraker**
- closes #14

### CODE REVIEW

File: app/(base-layout)/support/_component.tsx/contact.form.tsx [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+            rightSection={
+              contactForm.values.message && (
+                <TrashIcon
+                  onClick={() => contactForm.setFieldValue('message', '')}
+                />
+              )
+            }
+            {...contactForm.getInputProps('message')}
+          />
+          <p
```
Comment (BaptisteLegat) [inline app/(base-layout)/support/_component.tsx/contact.form.tsx:133]: use Text component

File: emails/utils/EmailLayout.tsx [hunk 2]
```diff
@@ -52,7 +52,7 @@ export const EmailLayout = ({ children }: PropsWithChildren) => {
               className="inline"
               alt={`${SiteConfig.company.name}'s logo`}
             />
-            <Link href={`${baseUrl}/api/emails/unsubscribe`}>Unsubscribe</Link>
+            <Link href={`${baseUrl}/api/emails/unsubscribe`}> Unsubscribe</Link>
```
Comment (BaptisteLegat) [inline emails/utils/EmailLayout.tsx:55]: Why ?

File: emails/utils/EmailLayoutWithoutFooter.tsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+import { Tailwind } from '@react-email/tailwind';
+import type { PropsWithChildren } from 'react';
+
+export const EmailLayoutWithoutFooter = ({ children }: PropsWithChildren) => {
+  const baseUrl = getServerUrl();
+  return (
+    <Tailwind
+      config={
+        {
+          // Theme can go here
```
Comment (BaptisteLegat) [inline emails/utils/EmailLayoutWithoutFooter.tsx:20]: add theme

File: emails/utils/EmailLayout.tsx (hunk 2 above)
Comment (Dercraker) [reply emails/utils/EmailLayout.tsx:55]: For the format of email

File: emails/utils/EmailLayoutWithoutFooter.tsx (hunk 3 above)
Comment (Dercraker) [reply emails/utils/EmailLayoutWithoutFooter.tsx:20]: Where is located this theme ?

**Comment by BaptisteLegat** [comment]
Le design de la page contact ne colle pas avec le design du figma

File: emails/utils/EmailLayoutWithoutFooter.tsx (hunk 3 above)
Comment (BaptisteLegat) [reply emails/utils/EmailLayoutWithoutFooter.tsx:20]: for add theme, u need to import this : 
`import tailwindConfig from '../../tailwind.config';
`

and just implement like that: 

```
    <Tailwind
      config={tailwindConfig}
    >
```
-> add colors like that into the .js file : 
```
primary: {
          50: '#ecfef8',
          100: '#d8faf0',
          200: '#acf7df',
          300: '#7ef4ce',
          400: '#5cf1c0',
          500: '#4bf0b6',
          600: '#40efb1',
          700: '#34d59b',
          800: '#27bd89',
          900: '#0aa374',
        },
```

I tried to use the theme but colors cant be apply on email (maybe need to use style)

**Review by Baptiste-Ferrand — APPROVED** [review]

File: app/(base-layout)/support/_component.tsx/contact.form.tsx
Comment (BaptisteLegat) [inline app/(base-layout)/support/_component.tsx/contact.form.tsx:1]: probleme sur le nom de dossier

**Pull request merged**